README.md

```diff
@@ -1 +1,22 @@
-# line-protocol
\ No newline at end of file
+# line-protocol
+
+This is an encoder for the influx line protocol. https://docs.influxdata.com/influxdb/v1.5/write_protocols/line_protocol_reference/
```

Comment:
Should be `latest` instead of `v1.5` in the URL. Maybe change that to an inline link rather than a simple URL out in the text.